src/tests/hook-tests.cpp

```diff
@@ -33,5 +34,20 @@ MAKE_HOOK_WRAPPER(test2_hook, &test2, bs_hook::Il2CppWrapperType, bs_hook::Il2Cp
     return ret;
     // Return from overall hook is converted to a void*
 }
+
+void install_a_hook() {
+    ::Hooking ::InstallHook<Hook_test>(il2cpp_utils ::Logger).after("test-mod");
+
+    modloader::ModInfo chroma = { "chroma", "1.0.0", 0 };
+    // reinstall after setting priorities   
+    auto& hook = INSTALL_HOOK(il2cpp_utils::Logger, test2_hook).after(chroma).before("test").reinstall();
+
+    if (false) {
```

Comment:
Why condition this out? Compiler could optimize this out and theoretically avoid even linting this